### DISCUSSION TITLE
Is there a check for emotes locked behind server boost levels?

### DISCUSSION THREAD

**Discussion by SuperZoops** [Rapptz/discord.py#6627] [Q&A] [answered]
As a stupid example, if I wanted to set up a listener that checks if an emote sent is available with the current amount of boosts.
If we lose level 2, we lose 50 emote slots.

**Comment by NCPlayz** [comment, marked as answer]
`discord.Emoji` has an [`.available`](https://discordpy.readthedocs.io/en/stable/api.html#discord.Emoji.available) attribute to be able to determine exactly that - whether or not it can be used with the main reason that can affect it being the server boost level. 

You should be able to check the [`Guild.premium_tier`](https://discordpy.readthedocs.io/en/stable/api.html#discord.Guild.premium_tier) to see if the boost level has dropped, as well as compare [`Guild.emoji_limit`](https://discordpy.readthedocs.io/en/stable/api.html#discord.Guild.emoji_limit), with the [`on_guild_update`](https://discordpy.readthedocs.io/en/stable/api.html#discord.on_guild_update) event.

**Reply by SuperZoops** [reply]
Thank you very much!
